### PR TITLE
fix: replace broken link in pull request template

### PR DESCRIPTION
**Pull request by jpwhitemn**
Signed-off-by: Jim White <jpwhite_mn@yahoo.com>

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-examples/blob/master/.github/CONTRIBUTING.md.

## What is the current behavior?

broken link in PR template

## What is the new behavior?
fixed link in PR template

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Specific Instructions
Are there any specific instructions or things that should be known prior to reviewing?

## Other information

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -0,0 +1,26 @@
+github.com/BurntSushi/toml v0.3.1 h1:WXkYYl6Yr3qBf1K79EBnL4mak0OimBfB0XUf9Vl28OQ=
```
Comment (lenny-intel) [inline go.sum:1]: go.sum shouldn't be committed. Should be in the .gitignore file.

**Review by lenny-goodell — CHANGES_REQUESTED** [review]

File: go.sum (hunk 1 above)
Comment (jpwhitemn) [reply go.sum:1]: hmmm... not sure how that go there.  fixing.

**Review by lenny-goodell — CHANGES_REQUESTED** [review]
go.sum is still in the PR. Try deleting it and push chnages.

**Review by lenny-goodell — APPROVED** [review]
LGTM

**Pull request merged**